pre-commit-config.yaml

```diff
@@ -75,3 +75,9 @@ repos:
         # darker is not compatible with v22.1.0
         # https://github.com/akaihola/darker/issues/264
         - black==21.12b0
+
+- repo: https://github.com/asottile/blacken-docs
+  rev: v1.12.1
+  hooks:
+    - id: blacken-docs
+      additional_dependencies: [black==22.10.0]
```

Comment:
Seems this is depending on a different version of black than defined above? We should add a comment explaining why that is, if we're doing it for some reason.

Reply:
I think each hook is allocated its own virtualenv?

There is a comment for the `darker` hook about `black==21.12b0` -- it has issues with `darker`, but that comment wouldn't apply for `blacken-docs`, so I guess they can be allowed to drift as long as the `darker` hook has this issue? Once the issue is solved, they can align?